ideahacks/routes/admin/parts.js

```diff
@@ -7,6 +7,28 @@ const getParts = (req, res) => {
 }
 
 const postParts = (req, res) => {
+  //return failure if anything is wrong with the submitted part
+  if (
+    req.body.partName === '' ||
+    !(Number.isInteger(req.body.stock) && req.body.stock >= 0) ||
+    req.body.description === '' ||
+    req.body.owners === undefined
+  ) {
+    return res.json({ status: 'failure', message: 'Please fill out all fields of the form!' })
+  }
+  //end of form failure test
```

Comment:
Actually can you remove this entire check? If we simply give the <input> field a "required" property, the browser will force the user to fill out all the required fields of the form.
Example: `<input class='part-name' type='text' name='partName' required>`

So remove this check and add "required" to each of the form input fields.